docs/02-guides/analytics/getting-started.mdx

```diff
@@ -0,0 +1,188 @@
+        // Settings can be an object or a function that will be called
+        // with the current consent authorization from the visitor, it also
+        // receive a 2nd parameter with codes of the other consents given.
+        // Using a function can allow to have different settings depending
+        // on the context
+        settings: (authorization, otherAuthorizations) => {
+          return {
+            measurementIds: ["G-abc123"],
+            gtagConfig: {
+              anonymize_ip: !authorization,
+            },
+          };
+        },
+        // It should either return a promise function or a dynamic import to
+        // the plugin script which will be added at to  the `analytics` library
+        // see listed plugins: https://getanalytics.io/plugins/#supported-analytic-tools
+        script: () => import("@analytics/google-analytics"),
+      },
+    ],
+  } satisfies AnalyticsConfig,
+};
+```
+
+### Retrieve authorized cookie services in analytics
+
+In Front-Commerce, to allow authorized cookie services to be injected in
+analytics services `settings(authorization, otherAuthorizations)` callback (as
+its 2nd parameter), those services also need to be defined in
+`config/analytics.js`.
+
+This can be done by declaring "dummy" modules in `config/analytics.ts` such as:
+
+```diff title="app/config/analytics.ts"
+       // ...
++      {
++        name: "my-service",
++        needConsent: true,
++        script: () => () => {
++          return {
++            name: "my-service",
++          };
++        },
++      },
+     ],
+   },
+ };
+```
+
+## The GDPR consent
+
+If your plugins need consent of the user before running, you need to setup the
+`cookiesServices.js` file. This file will let you define which cookies and
+trackings services are used within your application and will let the user chose
+which tracking service to allow.
+
+```js title="config/cookiesServices.js"
+export default {
+  default_en: [
+    {
+      // Category of cookies to allow the user to accept all the plugins at once in a specific category
+      title: "Analytics",
+      description:
+        "These cookies allows us to measure the traffic on our contents and hence to improve them.",
+      services: [
+        {
+          // The name should be the same as mentioned in the `config/analytics.js` file
+          name: "google-analytics",
+          title: "Google Analytics",
+          // display all the cookies managed by Google Analytics
+          cookies: [
+            "_ga",
+            "_gat",
+            "_gid",
+            "__utma",
+            "__utmb",
+            "__utmc",
+            "__utmt",
+            "__utmz",
+          ],
+          description:
+            "Google Analytics cookies, from Google, are meant to gather statistics about visits.",
+          link: "https://support.google.com/analytics/answer/6004245",
+        },
+      ],
+    },
+  ],
+};
+```
+
+The consent for the cookies is stored in 2 cookies:
```

Comment:
I'm not sure I understand your suggestion?

Reply:
It's just that "cookies" is mentionned two times in a 10-words sentence, I was wondering if it could be paraphrased to avoid that (but I'm not sure that it can).

Reply:
I rephrased it a bit.